recipes/buttons/callbar_button/callbar_button.vue

```diff
@@ -192,6 +192,13 @@ export default {
       }
       return this.circle ? 'outlined' : 'clear';
     },
+
+    callbarButtonListeners () {
+      return {
+        ...this.$listeners,
+        click: (event) => this.$emit('click', event),
+      };
+    },
```

Comment:
Hmm I am confused by this. We are now capturing an click from the tooltip wrapper, it should be captured from the button right? since that is the interactive element.

Reply:
Hmm yeah I think we should do that, it's the correct way to do it.

Reply:
Just moved the events to button, vue2 and vue3 tests were out of sync too.